src/decorators.ts

```diff
@@ -48,4 +50,42 @@ export function Exclude(options?: ExcludeOptions) {
         const metadata = new ExcludeMetadata(object instanceof Function ? object : object.constructor, propertyName, options || {});
         defaultMetadataStorage.addExcludeMetadata(metadata);
     };
+}
+
+/**
+ * Transform the object from class to plain object and return only with the exposed properties.
+ */
+export function TransformClassToPlain(params?: ClassTransformOptions): Function {
+
+    return function (target: Function, propertyKey: string, descriptor: PropertyDescriptor) {
+        const classTransformer: ClassTransformer = new ClassTransformer();
+        const MethodTransformer: Function = classTransformer.classToPlain;
```

Comment:
`MethodTransformer` is redundant here. Just call `classTransformer.classToPlain` below.